src/stylesheets/settings/_settings-components.scss

```diff
@@ -80,16 +80,14 @@ $theme-footer-max-width: "desktop" !default;
 // Form and input
 $theme-checkbox-border-radius: "sm" !default;
 $theme-form-font-family: "ui" !default;
+$theme-input-background-color: default !default;
```

Comment:
I changed this from a tile-specific background color to an input background color since this will now apply to original checkbox and radios as well